lambda/models/lambda_functions.py

```diff
@@ -88,139 +83,131 @@ async def model_already_exists_handler(request: Request, exc: ModelAlreadyExists
 
 @app.post(path="", include_in_schema=False)  # type: ignore
 @app.post(path="/")  # type: ignore
-async def create_model(create_request: CreateModelRequest, request: Request) -> CreateModelResponse:
+async def create_model(create_request: CreateModelRequest) -> CreateModelResponse:
     """Endpoint to create a model."""
-    headers = request.headers
     create_handler = CreateModelHandler(
-        base_uri=get_lisa_serve_endpoint(),
-        headers=headers,
-        verify=get_cert_path(iam_client),
+        stepfunctions_client=stepfunctions,
+        model_table_resource=model_table,
+        litellm_client=litellm_client,
     )
     return create_handler(create_request=create_request)
 
 
 @app.get(path="", include_in_schema=False)  # type: ignore
 @app.get(path="/")  # type: ignore
-async def list_models(request: Request) -> ListModelsResponse:
+async def list_models() -> ListModelsResponse:
     """Endpoint to list models."""
-    headers = request.headers
     list_handler = ListModelsHandler(
-        base_uri=get_lisa_serve_endpoint(),
-        headers=headers,
-        verify=get_cert_path(iam_client),
+        stepfunctions_client=stepfunctions,
+        model_table_resource=model_table,
+        litellm_client=litellm_client,
     )
     return list_handler()
 
 
 def _create_dummy_model(model_name: str, model_type: ModelType, model_status: ModelStatus) -> LISAModel:
```

Comment:
I think we can delete this now right?

Reply:
still used as placeholders for the update/start/stop api's at the moment